<launch>
    <node pkg="cyglidar_d1" type="cyglidar_pcl_publisher" name="line_laser" output="screen">
    <param name="port" value="/dev/cyglidar"/>
    <param name="baud_rate" value="3000000"/>
    <param name="version_num" value="2"/>
    <param name="frame_id" value="laser_link"/>
    <param name="fixed_frame" value="/map"/>
    </node>

    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find cyglidar_d1)/rviz/cyglidar_config.rviz"/>
</launch>
